packages/owork/owork.0.1.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "@doc"] {with-doc}
+]
+maintainer: ["Jeffas <dev@jeffas.io>"]
+authors: ["Jeffas"]
+bug-reports: "https://github.com/jeffa5/owork/issues"
+homepage: "https://github.com/jeffa5/owork"
+doc: "https://jeffa5.github.io/owork"
+dev-repo: "git+https://github.com/jeffa5/owork.git"
+synopsis: "A productivity timer for focusing on work"
+description:
+  "Owork is a productivity timer. It provides a server executable which uses a state machine to start different timers for sessions. There are four states: `idle`, `working`, `short break` and `long break`."
+depends: [
+  "dune" {build & > "1.5"}
```

Comment:
```suggestion
  "dune" {>= "1.10"}
```
Dune should not be tagged as a build dependency and according to your `dune-project` file, this package requires at least dune 1.10